tests/hypothesis_strategies/parameters.py

```diff
@@ -25,13 +25,13 @@
     st.booleans(),
     st.floats(min_value=0.0, max_value=1.0, exclude_min=True, exclude_max=True),
 )
-"""A strategy that generates decorrelation settings."""
+"""A recommender that generates decorrelation settings."""
```

Comment:
In all of the hypothesis files we want the word strategy. Will not mark this everywhere but only here.

Reply:
I would say we prob want neither  to avoid confusion with other occurrences of `strategy`, why not call them just `hypothesis` ?

Reply:
I changed that back for now, but overall I think the term `strategy` could and should also be avoided in hypothesis